lib/topics.js

```diff
@@ -8,18 +8,37 @@ const SNS    = require('./sns');
 class Topic {
 
   constructor(server, topicName) {
-    this._server = server;
-    this.name    = topicName;
+    this._server     = server;
+    this.name        = topicName;
+    this.subscribers = [];
```

Comment:
I'm not 100% sure I follow, but are you suggesting that it'd be more useful if Ironium threw an error if the handler is already in the set?